src/components/Books.js

```diff
@@ -0,0 +1,34 @@
+import Book from './Book';
+import styles from '../styles/Books.module.css';
+
+const Books = () => (
+  <div className={styles.books}>
+    <ul className={styles.list}>
+      <li className={styles.item}>
+        <Book title="A Game of Thrones" author="George R. R. Martin" />
+      </li>
+      <li className={styles.item}>
+        <Book title="Ego is the Enemy" author="Ryan Holiday" />
+      </li>
+      <li className={styles.item}>
+        <Book title="Fire and Blood" author="George R. R. Martin" />
+      </li>
+    </ul>
```

Comment:
- [ ] It would be nice if you make an array of objects for the books and display that array of objects in this Books component using a map which is a professional approach and you don't need to write the same lines again and again. Other than that, suppose you have 10000 books then what do you think are you gonna write 10000 lines here :thinking: obviously no. You need an approach such as an iteration which will iterate a list of books which is shown below :+1:

```jsx
const Books = () => {
  const books = [
    {title: "A Game of Thrones", Author: "George R. R. Martin"},
    {title: "Ego is the Enemy", Author: "Ryan Holiday"},
    {title: "Fire and Blood", Author: "George R. R. Martin"},
  ];
  return (
    <div className={styles.books}>
        <ul className={styles.list}>
    {
    books.map(book =>(
      <li className={styles.item}>
        <Book title={book.title} author={book.author} />
      </li>
    ))
    }
    </ul>
    </div>
  )
}
```

Reply:
# Thanks for your wonderful insight. But I think the activity requirement is just to setup the project.

Reply:
Hey, @roniy68 I hope you are doing well. You are right dear. You can leave this requirement and work on others :blush:

Reply:
![image](https://user-images.githubusercontent.com/93039359/221482692-960aa369-e9c4-4da4-bd7a-248d6dd528a8.png)

Hey @roniy68 I hope you are doing well :+1: This is a requirement in your project that you need to display a list of books and you know when you talk about displaying a list then it means you are rendering an array of objects :+1: which you have done so many times so far :+1:

Reply:
Thanks @HAMMAS-SALEEM I have fixed as you suggested :-)
Happy Coding 🔢